### ISSUE TITLE
Validation errors on synthetichealth.github.io/synthea/ extensions

### ISSUE THREAD

**Issue by gotdan** [synthetichealth/synthea#1437]
### What happened?

The HL7 FHIR validator is reporting errors related to the `http://synthetichealth.github.io/synthea` extensions. E.g.,
- Patient.extension[5]: Error - The extension http://synthetichealth.github.io/synthea/disability-adjusted-life-years could not be found so is not allowed here
- Patient.extension[6]: Error - The extension http://synthetichealth.github.io/synthea/quality-adjusted-life-years could not be found so is not allowed here
- Practitioner.extension[0]: Error - The extension http://synthetichealth.github.io/synthea/utilization-encounters-extension could not be found so is not allowed here

### Environment

```markdown
- OS:
- Java:
```

**Comment by jawalonoski** [comment]
The first two URLs (QALYs and DALYs) resolve to FHIR StructureDefinitions for those extensions.

The third link (encounter utilization) is broken.

My understanding was the that resolvable computable extension definitions was a SHOULD, not a SHALL.

What is the requirement for an extension to be "found"?

**Comment by jawalonoski** [comment]
Given this thread https://chat.fhir.org/#narrow/stream/291844-FHIR-Validator/topic/Extension.20could.20not.20be.20found/near/435291152 it sounds like there is an impasse.

I suppose we could post the StructureDefinition for the utilization-encounters-extension.

**Comment by gotdan** [comment]
Yup, agree that apart from the missing StructureDefinition (which would be nice to have :), this seems more like a validator configuration issue than a Synthea issue. I'll open a ticket on https://github.com/inferno-framework/bulk-data-test-kit to support allowing unknown extensions during validation and/or passing in valid StructureDefinition urls.
